Sprint-2/stretch/till.js

```diff
@@ -8,24 +8,44 @@ function totalTill(till) {
   let total = 0;
 
   for (const [coin, quantity] of Object.entries(till)) {
-    total += coin * quantity;
+
+    let coinValue;
+
+    if(coin.endsWith("p")){
+      coinValue = parseInt(coin.replace('p', ""), 10);
+
+    } else if (coin.startsWith("£")){
+      throw new Error("Invalid coin format");
+    }
```

Comment:
Among all possible invalid coin formats (e.g., "$1.00", "Foobar") , why specifically check for this particular invalid coin format?